04-networking.md

```diff
@@ -75,11 +75,11 @@ The following two resource groups will be created and populated with networking
    > :book: The networking team receives a request from an app team in business unit (BU) 0001 for a network spoke to house their new AKS-based application (Internally know as Application ID: A0008). The network team talks with the app team to understand their requirements and aligns those needs with Microsoft's best practices for a general-purpose AKS cluster deployment. They capture those specific requirements and deploy the spoke, aligning to those specs, and connecting it to the matching regional hub.
 
    ```bash
-   RESOURCEID_VNET_HUB=$(az deployment group show -g rg-enterprise-networking-hubs -n hub-default --query properties.outputs.hubVnetId.value -o tsv)
-   echo RESOURCEID_VNET_HUB: $RESOURCEID_VNET_HUB
+   export RESOURCEID_VNET_HUB_AKS_BASELINE=$(az deployment group show -g rg-enterprise-networking-hubs -n hub-default --query properties.outputs.hubVnetId.value -o tsv)
```

Comment:
Why are we exporting now?

Reply:
because https://github.com/mspnp/aks-baseline/blob/main/saveenv.sh save the variables that are exported and end by _AKS_BASELINE. 
Without the export, the variables are not saved. 
That is good when you for some reason need to re-start a deploy process in the middle, you have the variable values till the moment you executed the latest  saveenv.sh.

Reply:
Talked over teams, plan is to revert.